### ISSUE TITLE
Feature Request: Minimum entries before selection

### ISSUE THREAD

**Issue by VasVadum** [jagrosh/GiveawayBot#97]
I would like to see the ability to set a minimum # or % of users who enter a give-away before the event can fire and select a random person. For example, I could set it for 2 weeks and >10%. Or perhaps 2 weeks and >20. If I type % in the command, it calculates the current members of that channel or server and sets it to that number. Then it must wait 2 weeks, and if less than that number has entered, it waits till that number has been reached or cancels the giveaway.

**Comment by jagrosh** [comment]
Currently, the bot doesn't receive the reactions until the end of the giveaway. This feature may be possible (and possibly easier) after the rewrite, which will use interactions.

**Comment by Kergh** [comment]
Yeah minimum entries before selection a winner had been super nice :)

**Comment by IAMTHEGOAT123333** [comment]
this app is good

**Comment by IAMTHEGOAT123333** [comment]
it is so good

**Comment by Kergh** [comment]
Yeah a minimum amount of entries Before the bot select a winner had been super nice 👍
